### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
- align README.rst, Settings.cfg, Index.rst, Includes.rst.txt, genindex.rst
- move Sitemap/Index.rst to Sitemap.rst
- remove outdated Targets.rst
- fix rendering warnings
- fully embed new file structure page

Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/181

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
Leftovers:

* Documentation/GeneralConventions/Glossary.rst: how to handle introduction of outdated :pn: text role here, @marble ?

File: Documentation/Index.rst [hunk 1]
```diff
@@ -1,11 +1,36 @@
 .. include:: /Includes.rst.txt
+
 .. index:: ! Writing documentation
-.. _start:
 
 =====================
 Writing documentation
 =====================
 
+:Version:
```
Comment (linawolf) [inline Documentation/Index.rst:9]: I would put this part below the cards with the deep links. All information can be gained from the left as well and the cards are necessary for a quick navigation, while the other links will be seldomly clicked. Lets not lose the advantage we just gained here.

**Review by linawolf — CHANGES_REQUESTED** [review]

File: Documentation/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Index.rst:9]: Sure. Adjust to your likes.

Comment (alexander-nitsche) [reply Documentation/Index.rst:9]: It looks currently like this.
Screen Shot 2022-03-22 at 09 56 15

**Review by linawolf — APPROVED** [review]

**Pull request merged**